build.gradle

```diff
@@ -1239,6 +1239,9 @@ project(':core') {
       }
     }
   }
+  // -parameters generates arguments with parameter names in TestInfo#getDisplayName.
+  // ref: https://github.com/junit-team/junit5/blob/4c0dddad1b96d4a20e92a2cd583954643ac56ac0/junit-jupiter-params/src/main/java/org/junit/jupiter/params/ParameterizedTest.java#L161-L164
+  compileTestScala.options.compilerArgs.add "-parameters"
```

Comment:
Instead of adding the args to core module directly, could we define the compile args in `ScalaCompile`? we can add condition check to make sure the `-parameters` is added to test only. For example:

```
    if (name == "compileTestScala")
      options.compilerArgs << "-parameters"
```

Reply:
Rework as you suggested.